[hi] Localized policy-as-code.md

oops-shlok opened the pull request:
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

oops-shlok committed:
[hi] Localized policy-as-code.md

Signed-off-by: Shlok Upadhyay <deep.upu9@gmail.com>

aj11anuj reviewed (changes requested):
Same mistake, [refer this comment](https://github.com/cncf/glossary/pull/3443#pullrequestreview-2760283037)

oops-shlok committed:
[hi] Fixed Core hindi words

Signed-off-by: Shlok Upadhyay <deep.upu9@gmail.com>

oops-shlok commented:
> Same mistake, [refer this comment](https://github.com/cncf/glossary/pull/3443#pullrequestreview-2760283037)

I have fixed it now and updated the code. Please review once.

aj11anuj reviewed (commented):
Refer [comment](https://github.com/cncf/glossary/pull/3456#pullrequestreview-2850509299), you have used words like "nitiyan", "dastavez", "machine-pathniye" like traditional wordings here. Do give it a read and change the wordings to make it sound more natural.

mailnike reviewed (commented):
Thanks for this localization, it reads clearly and the technical content is accurate. Minor suggestion: the title `पॉलिसी ऐज़ कोड (Policy as Code (PaC))` has nested parentheses. Consider `पॉलिसी ऐज़ कोड (Policy as Code, PaC)` for a cleaner look consistent with other terms.
